pom.xml

```diff
@@ -209,6 +209,12 @@
       <version>2.5</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Indeed, and I may even use it in some projects at work. However, other commons components use - as far as I can remember - JUnit with some local helper classes for tests. So my line of thinking, is that even though it looks useful, others would have to learn and remember to use it, or eventually we would end up with calls to Equalsverifier methods, and in some other places the old way. But again, -0 is not a blocker, if others prefer to have it, I'm fine with it :-)